JYP-iOS/JYP-iOS/Sources/CompositionRoot.swift

```diff
@@ -0,0 +1,124 @@
+            UINavigationController(rootViewController: myPageViewController)
+        ]
+
+        return viewController
+    }
+    
+    static func makeMyPlannerScreen() -> MyPlannerViewController {
+        let journeyService: JourneyServiceType = JourneyService(provider: ServiceProvider.shared)
+        
+        let pushPlannerInviteScreen: (_ id: String) -> PlannerInviteViewController = { (id) in
+            let reactor = PlannerInviteReactor(id: id)
+            let controller = PlannerInviteViewController(reactor: reactor)
+            
+            return controller
+        }
+        
+        let pushPlannerRouteScreen: (_ root: AnyObject.Type, _ journey: Journey, _ order: Int) -> PlannerRouteViewController = { (root, journey, order) in
+            let reactor = PlannerRouteReactor(journey: journey, order: order, journeyService: journeyService)
+            let controller = PlannerRouteViewController(reactor: reactor, root: PlannerViewController.self)
+            
+            return controller
+        }
+        
+        let pushPlannerScreen: (_ id: String) -> PlannerViewController = { (id) in
+            let reactor = PlannerReactor(id: id)
+            let controller = PlannerViewController(reactor: reactor,
+                                                   pushPlannerInviteScreen: pushPlannerInviteScreen,
+                                                   pushPlannerRouteScreen: pushPlannerRouteScreen)
+            
+            return controller
+        }
+        
+        let pushInputPlannerCodeBottomSheetScreen: () -> InputPlannerCodeBottomSheetViewController = {
+            let reactor = InputPlannerCodeBottomSheetReactor()
+            let controller = InputPlannerCodeBottomSheetViewController(reactor: reactor,
+                                                                       pushPlannerInviteScreen: pushPlannerInviteScreen)
+            
+            return controller
+        }
+        
+        let pushSelectionPlannerJoinBottomScreen: () -> SelectionPlannerJoinBottomViewController = {
+            let controller = SelectionPlannerJoinBottomViewController(mode: .drag,
+                                                                      pushInputPlannerCodeBottomSheetScreen: pushInputPlannerCodeBottomSheetScreen)
+            
+            return controller
+        }
+        
+        let reactor = MyPlannerReactor()
+        let viewController = MyPlannerViewController(reactor: reactor,
+                                                     pushSelectionPlannerJoinBottomScreen: pushSelectionPlannerJoinBottomScreen,
+                                                     pushPlannerScreen: pushPlannerScreen)
+        let tabBarItem = UITabBarItem(title: nil, image: JYPIOSAsset.myJourneyInactive.image.withRenderingMode(.alwaysOriginal), selectedImage: JYPIOSAsset.myJourneyActive.image.withRenderingMode(.alwaysOriginal))
```

Comment:
줄바꿈이 조금 이상하게 되어있는 것 같은데 수정 부탁드립니다~
```suggestion
        let tabBarItem = UITabBarItem(
            title: nil,
            image: JYPIOSAsset.myJourneyInactive.image.withRenderingMode(.alwaysOriginal),
            selectedImage: JYPIOSAsset.myJourneyActive.image.withRenderingMode(.alwaysOriginal)
        )
```

Reply:
107, 117 line도 동일합니다.

Reply:
수정했습니다 !